src/app/core/models/server-config/server-config.model.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * model for config data (parameters, services etc.) retrieved from the ICM server.
+ */
+
+export interface ServerConfig {
+  [key: string]: string | boolean | string[] | ServerConfig;
```

Comment:
for this feature such a complex data structure is not needed?

Reply:
I think the recursive definition is easier here, see https://github.com/intershop/intershop-pwa/pull/179#discussion_r404208257